docs/product/components/topbar.html

```diff
@@ -0,0 +1,42 @@
+---
+layout: page
+title: Topbar
+description: A topbar component
+---
+
+<section class="stacks-section">
+    {% header "h2", "Logged in" %}
+
+    {% assign variations = "s-topbar__auto,s-topbar__dark,s-topbar__light" | split: "," %}
```

Comment:
Is this here 'cause you don't wanna pass an empty class or something? `s-topbar__auto` is the same as just a plain ol' `s-topbar` yeah? Think you can just kill it.

Reply:
More or less. I show the class above the generated example right now, but that might change once the docs are actually written. In that case, we can make this an empty class or whatever really.